setup.py

```diff
@@ -19,7 +19,7 @@
 
 
 install_requires = [
-    "Django>=3.2,<4.3",
+    "Django>=4.2",
```

Comment:
Per the discussion with the core team, I'm going to add the upper bound but with `<6.0` instead of `<5.1`